dask/dataframe/io/csv.py

```diff
@@ -50,6 +50,8 @@ def pandas_read_text(reader, b, header, kwargs, dtypes=None, columns=None,
         A dictionary of keyword arguments to be passed to ``reader``
     dtypes : dict
         DTypes to assign to columns
+    path: tuple
```

Comment:
nit: space needed before the colon for "proper" numpydoc formatting (see https://numpydoc.readthedocs.io/en/latest/format.html)

>  The colon must be preceded by a space, or omitted if the type is absent.